ci/hera_cal_tests.yml

```diff
@@ -29,5 +28,5 @@ dependencies:
     - git+https://github.com/HERA-Team/linsolve
     - git+https://github.com/RadioAstronomySoftwareGroup/pyuvsim
     - git+https://github.com/RadioAstronomySoftwareGroup/pyradiosky
-    - git+https://github.com/HERA-Team/uvtools
+    - git+https://github.com/HERA-Team/uvtools.git@skip_wgt_clean_passing_fix
```

Comment:
I'd undo this.

Reply:
Will do this now that the uvtools fixes have been merged.